config/bot_config.yaml

```diff
@@ -1,8 +1,8 @@
 intervals:
     submission_interval_hours: 24
-    sleep_interval_seconds: 300
+    sleep_interval_seconds: 10
```

Comment:
This shouldn't have gotten changed, but it's not a big deal. The value is meant to be configurable.